python/social_bot/envs/grocery_ground.py

```diff
@@ -500,9 +497,13 @@ def __init__(self,
             icub_standing_task = ICubStandingTask()
             icub_aux_task_group.add_task(icub_standing_task)
             self._teacher.add_task_group(icub_aux_task_group)
+        self._teacher._build_vocab_from_tasks()
```

Comment:
This might need to be removed. _build_vocab_from_tasks is done within teacher.py automatically when "teacher.sentence_to_sequence()" is called

Reply:
I need the vocab_size before sentence_to_sequence() is called.  Unless I missed something, this call is needed?

Reply:
Sorry. I have not noticed that : D